src/foremast/s3/s3apps.py

```diff
@@ -189,6 +190,20 @@ def _put_bucket_encryption(self):
         LOG.debug('Response setting up S3 encryption: %s', _response)
         LOG.info('S3 encryption configuration updated')
 
+    def _put_bucket_lifecycle(self):
+        if self.s3props['lifecycle']['enabled']:
+            lifecycle_config = {'Rules': [{}]}
+            lifecycle_config = {
+                'Rules': self.s3props['lifecycle']['lifecycle_rules']
+            }
+            LOG.debug(lifecycle_config)
```

Comment:
You should prefix the debug with what it is...
`LOG.debug('Lifecycle Config: ....)`